src/participant/participant.ts

```diff
@@ -707,14 +742,39 @@ export default class ParticipantController {
       request,
       connectionNames: this._getConnectionNames(),
     });
-    const responseContentWithNamespace = await this.getChatResponseContent({
-      messages: messagesWithNamespace,
-      token,
-    });
-    const { databaseName, collectionName } =
-      Prompts.namespace.extractDatabaseAndCollectionNameFromResponse(
-        responseContentWithNamespace
-      );
+
+    let {
+      databaseName,
+      collectionName,
+    }: {
+      databaseName: string | undefined;
+      collectionName: string | undefined;
+    } = {
+      databaseName: undefined,
+      collectionName: undefined,
+    };
+
+    // When there's no user message content we can
+    // skip the request to the model. This would happen with /schema.
+    if (Prompts.doMessagesContainUserInput(messagesWithNamespace)) {
+      // VSCODE-626: When there's an empty message sent to the ai model,
+      // it currently errors (not on insiders, only main VSCode).
+      // Here we're defaulting to have some content as a workaround.
+      // TODO: Remove this when the issue is fixed.
+      messagesWithNamespace[messagesWithNamespace.length - 1].content =
+        messagesWithNamespace[
+          messagesWithNamespace.length - 1
+        ].content.trim() || 'see previous messages';
```

Comment:
How this can happen that we have user input but content is empty? And what are those 'see previous messages'? Which kind of messages does it refer to?

Reply:
An empty user message can happen when previous messages in the history have user input, then the most recent message is either empty, or the most recent user message is an empty one that is pulled from chat history. We get rid of some messages in the message building, any connection messages for instance, so this could be something from earlier in the chat history.